js/course_settings_disable_course_conclude.js

```diff
@@ -64,6 +66,7 @@ function initHUGlobal() {
         if ($target.not('.datepickerDisabled').hasClass('hasDatepicker')) {
           $target.addClass('datepickerDisabled');
           disableCourseConcludeDate();
+          addCourseConcludeDateDisabledMessage();
```

Comment:
I think we can avoid the double-call by adding a disconnect after we're done all our stuff in this code block:
```
classAttrObserver.disconnect();
```
That may obviate the need for the extra code in `addCourseConcludeDateDisabledMessage()` (although that extra code won't hurt!).